docs/geedocs/docsrc/answer/getStarted/overview.rst

```diff
@@ -257,7 +258,7 @@ Overview
       .. rubric:: Learn More
          :name: learn-more-9
 
-      -  :doc:`Fusion Administration Guide <../fusionAdministration>`
+      -  :doc:`Fusion Administration Guide <../fusionAdministration/beforeYouConfigure>`
```

Comment:
Why was this changed to "beforeYouConfigure"? I would expect to go to the top level of the guide  based on the link name

Reply:
this change was made to look like the previous versions. 
http://www.opengee.org/geedocs/5.3.1/answer/6353404.html

FusionAdministration link takes us to the  first section under fusionAdmin: http://www.opengee.org/geedocs/5.3.1/answer/3481464.html

Reply:
Before you configure is kind of confusing regarding this context. I'd rather have it go to the top level or first page or just remove it